serverless-functions/src/functions/features/park-interaction/flex/park-interaction.js

```diff
@@ -3,6 +3,9 @@ const { prepareFlexFunction, extractStandardResponse } = require(Runtime.getFunc
 ].path);
 const ConversationsOperations = require(Runtime.getFunctions()['common/twilio-wrappers/conversations'].path);
 const InteractionsOperations = require(Runtime.getFunctions()['common/twilio-wrappers/interactions'].path);
+const SyncClient = require('twilio-sync').Client;
+
+const syncToken = require(Runtime.getFunctions()['common/twilio-wrappers/get-sync-token'].path);
```

Comment:
Rather than using the Sync client, can we use Sync API via the Twilio client? That will avoid running into Sync client limits.

Reply:
If you look at the chat-to-video feature, there is an example of using Sync API via the Twilio client.

Reply:
Refactored this piece to use that sync wrapper, and deleted the `get-sync-token` function that I had created.